### PR TITLE
LPS-179688 As an Instance Administrator, I want to be able to configure that if the users have the option to give their password at registration+ minor SF

### PR DESCRIPTION
**Pull request by stian-sigvartsen**

### CODE REVIEW

**Comment by stian-sigvartsen** [comment]
ci:stop

**Comment by stian-sigvartsen** [comment]
ci:test:sf

**Comment by stian-sigvartsen** [comment]
ci:test:relevant

**Comment by JoyceWang08211** [comment]
ci:test:bundle

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -9578,6 +9578,7 @@ logger-name=Logger Name
 logging-audit-message-processor-configuration-name=Logging Message Audit Message Processor
 login=Login
 login-configuration-name=Login
+login-create-account-allow-custom-password=Allow Custom Password At Account Creation
```
Comment (hhtank) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:9581]: @thabet91 Just FYI, when we're making changes to match QA's verification result, we'll need to adjust the text here as well. As BChan's recommendation using https://capitalizemytitle.com, we'll put this text as `Allow Custom Password at Account Creation` instead.

**Comment by hhtank** [comment]
@stian-sigvartsen @JoyceWang08211 @thabet91 I'd like to close this PR since there are pending QA bugs that we need to figure out.

**Comment by hhtank** [comment]
ci:close

**Pull request closed without merging**